src/mame/atari/atarist.cpp

```diff
@@ -1605,8 +1605,8 @@ void stbook_state::stbook(machine_config &config)
 	WD1772(config, m_fdc, U517/2);
 	m_fdc->intrq_wr_callback().set(m_mfp, FUNC(mc68901_device::i5_w)).invert();
 	m_fdc->drq_wr_callback().set(FUNC(st_state::fdc_drq_w));
-	FLOPPY_CONNECTOR(config, WD1772_TAG ":0", atari_floppies, "35dd", 0, st_state::floppy_formats);
-	FLOPPY_CONNECTOR(config, WD1772_TAG ":1", atari_floppies, 0,      0, st_state::floppy_formats);
+	FLOPPY_CONNECTOR(config, WD1772_TAG ":0", atari_floppies, "35dd", 0, st_state::floppy_formats).enable_sound(true);
+	FLOPPY_CONNECTOR(config, WD1772_TAG ":1", atari_floppies, 0,      0, st_state::floppy_formats).enable_sound(true);
```

Comment:
I believe the 0 instead of `nullptr` for slot type won't compile, but this block is disabled under `#if 0`.